src/app/components/login-page/login-page.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { LoginService } from '../../services/login-service/login-service.service';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-login-page',
+  templateUrl: './login-page.component.html',
+  styleUrls: ['./login-page.component.scss']
+})
+export class LoginPageComponent implements OnInit {
+
+  usernameInput: string;
+  passwordInput: string;
+  loginMessage: string;
+
+  constructor(private loginService: LoginService, public router: Router) { }
+
+  sendLogin() {
+    if (this.usernameInput && this.passwordInput) {
+      this.loginService.login(this.usernameInput, this.passwordInput).subscribe((response) => {
+        if (response.statusCode === 200) {
```

Comment:
login service should not return with an entity that has status code. dont bind the function response to a http entity.

Reply:
Fixed, thanks, removed it alltogether.